README.md

```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
     return lhs.GUID == rhs.GUID
 }
 
+enum Result<T> {
+	case value(T)
+	case failure(Error)
+
+	func success(completionHandler: (T) -> ()) {
+		switch self {
+		case .value(let value):
+			completionHandler(value)
+		case .failure:
+			return
+		}
+	}
+
+	func failed(completionHandler: (Error) -> ()) {
+		switch self {
+		case .value:
+			return
+		case .failure(let error):
+			completionHandler(error)
+		}
+	}
+}
+
 struct FetchResponse {
-    let items: [Item]
-    let nextFetchTime: Int
+	let items: [Item]
+	let nextFetchTime: Int
 }
 
 protocol FetcherType {
-    func fetch() -> Result<FetchResponse>
+	func fetch() -> Result<FetchResponse>
 }
 
 struct Fetcher : FetcherType {
-    let domain: String
-
-    func randomItems() -> [Item] {
-        let items = [
-            Item(domain: domain, title: "Swift 2.0", GUID: "1"),
-            Item(domain: domain, title: "Strings in Swift 2", GUID: "2"),
-            Item(domain: domain, title: "Swift-er SDK", GUID: "3"),
-            Item(domain: domain, title: "Swift 2 Apps in the App Store", GUID: "4"),
-            Item(domain: domain, title: "Literals in Playgrounds", GUID: "5"),
-            Item(domain: domain, title: "Swift Open Source", GUID: "6")
-        ]
-        return [Item](items[0..<Int(arc4random_uniform(UInt32(items.count)))])
-    }
-
-    func fetch() -> Result<FetchResponse> {
-        if arc4random_uniform(2) == 0 {
-            let fetchResponse = FetchResponse(
-                items: randomItems(),
-                nextFetchTime: 300.milliseconds.fromNow()
-            )
-            return Result.Value(fetchResponse)
-        } else {
-            struct Error : ErrorProtocol, CustomStringConvertible { let description: String }
-            return Result.Error(Error(description: "Network Error"))
-        }
-    }
+	let domain: String
+
+	func randomItems() -> [Item] {
+		let items = [
+			Item(domain: domain, title: "Swift 2.0", GUID: "1"),
+			Item(domain: domain, title: "Strings in Swift 2", GUID: "2"),
+			Item(domain: domain, title: "Swift-er SDK", GUID: "3"),
+			Item(domain: domain, title: "Swift 2 Apps in the App Store", GUID: "4"),
+			Item(domain: domain, title: "Literals in Playgrounds", GUID: "5"),
+			Item(domain: domain, title: "Swift Open Source", GUID: "6")
+		]
+		return [Item](items[0..<Int(arc4random_uniform(UInt32(items.count)))])
+	}
+
+	func fetch() -> Result<FetchResponse> {
+		if arc4random_uniform(2) == 0 {
+			let fetchResponse = FetchResponse(
+   items: randomItems(),
+   nextFetchTime: Int(300.milliseconds.fromNow())
+   )
+			return Result.value(fetchResponse)
+		} else {
+			struct LocalError : Error, CustomStringConvertible { let description: String }
+			return Result.failure(LocalError(description: "Network Error"))
+		}
+	}
 }
 
 protocol SubscriptionType {
-    var updates: ReceivingChannel<Item> { get }
-    func close() -> ErrorProtocol?
+	var uprint_headerates: ReceivingChannel<Item> { get }
```

Comment:
Looks to me like it's still there? I was referring to `upring_headerates` to `updates`.